cmd/cloak/serve.go

```diff
@@ -0,0 +1,10 @@
+//go:build !linux
```

Comment:
Random thought: let's make a follow up issue for figuring out how to run tests on different platforms. It would be great to eventually have a test where we run dagger on macos and verify we can start daggerd up in a linux vm on there.

That's extremely not straightforward to automate (I think? would love to be wrong) so we can't do it right now, but it's also extremely important in that it's both a very common use case and a very complicated one where lots could go wrong.

Reply:
Ended up making an issue for this here: https://github.com/dagger/dagger/issues/3225